lib/polyfill/storage_estimate.js

```diff
@@ -0,0 +1,54 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.polyfill.StorageEstimate');
+
+goog.require('shaka.polyfill');
+
+
+/**
+ * @summary A polyfill to provide navigator.storage.estimate in old
+ * webkit browsers.
+ */
+shaka.polyfill.StorageEstimate = class {
+  /**
+   * Install the polyfill if needed.
+   */
+  static install() {
+    if ('storage' in navigator && 'estimate' in navigator.storage) {
+      // No need.
+      return;
+    }
+
+    if ('webkitTemporaryStorage' in navigator &&
+        'queryUsageAndQuota' in navigator.webkitTemporaryStorage) {
+      if (!('storage' in navigator)) {
+        navigator.storage = /** @type {StorageManager} */ {};
+      }
+      navigator.storage.estimate =
+        shaka.polyfill.StorageEstimate.storageEstimate_;
```

Comment:
I think this needs to be indented 2 more spaces.  Did you run `./build/check.py` or `./build/all.py`?

Reply:
Fixed!